api/api.go

```diff
@@ -1362,3 +1363,24 @@ func (s *ProxySpec) IsPureImport() bool {
 
 	return (s.PureBlockSpec != nil && s.PureBlockSpec.FullVolName != "") || (s.PureFileSpec != nil && s.PureFileSpec.FullVolName != "")
 }
+
+func (s *ProxySpec) GetPureFullVolumeName() string {
+	if !s.IsPureImport() {
+		return ""
+	}
+
+	if s.PureBlockSpec != nil {
+		return s.PureBlockSpec.FullVolName
+	}
+
+	if s.PureFileSpec != nil {
+		return s.PureFileSpec.FullVolName
+	}
+
+	return ""
+}
+
+// GetAllEnumInfo returns an EnumInfo for every proto enum
+func GetAllEnumInfo() []protoimpl.EnumInfo {
+	return file_api_api_proto_enumTypes
+}
```

Comment:
This function is not present in the commit that is being cherry-picked. How did this get added?

Reply:
That is... a good question, I don't know. I'll fix up the commit.

Reply:
Fixed. I remember having weird issues with that method the first time I added this method too a few months ago, so I dunno what's up with that.